### DISCUSSION TITLE
Transmission-daemon 3.00 tr_crypto_utils OpenSSL error on Ubuntu

### DISCUSSION THREAD

**Discussion by Adler-lug** [transmission/transmission#2713] [Q&A]
Install package **[transmission-daemon 3.00-1ubuntu1~20.04.1](https://launchpad.net/~transmissionbt/+archive/ubuntu/ppa)** on _Ubuntu Server 20.04_ and have such a problem in the log: 

> Nov 15 17:24:42 HOME-SERVER transmission-daemon[3092]: [2020-11-15 17:24:42.322] tr_crypto_utils OpenSSL error: error:05066066:Diffie-Hellman routines:compute_key:invalid public key (crypto-utils-openssl.c:387)
> Nov 15 17:24:43 HOME-SERVER transmission-daemon[3092]: [2020-11-15 17:24:43.322] tr_crypto_utils OpenSSL error: error:05066066:Diffie-Hellman routines:compute_key:invalid public key (crypto-utils-openssl.c:387)
> Nov 15 17:24:44 HOME-SERVER transmission-daemon[3092]: [2020-11-15 17:24:44.319] tr_crypto_utils OpenSSL error: error:05066066:Diffie-Hellman routines:compute_key:invalid public key (crypto-utils-openssl.c:387)

Otherwise, I don't see any problems visually, but the log grows.
The transmission is installed on a freshly installed ubuntu (updated the version of the operating system on the home server and installed the latest versions).
Tell me what could be the problem and how to solve it?

**Comment by LHozzan** [comment]
Unfortunately, I can confirm this bug also on **Debian Bullseye (11)** too.
```
apt-cache show transmission
Package: transmission
Version: 3.00-1
Installed-Size: 9
Maintainer: Sandro Tosi <morph@debian.org>
Architecture: all
```

I fully agree with @Adler-lug . From my perspective, this problem occurring totally random, I not see any correlation.

Supporting the Transmission all another BT clients? Occurring this problem only when that unsupported client attempt to connect to my client? I am not sure, maybe I am wrong, just a speculation.

**Comment by ckerr** [comment]
This appears to be a configuration issue rather than a Transmission bug, so I'm converting this to a discussion thread.

**Reply by 1o1o1** [reply]
FreeBSD port too :)

transmission-daemon[52335]: tr_crypto_utils OpenSSL error: error:05066066:Diffie-Hellman routines:compute_key:invalid public key (/usr/ports/net-p2p/transmission-daemon/work/transmission-3.00/libtransmission/crypto-utils-openssl.c:387)
